teleport-server/instance.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_instance" "teleport_instance" {
+  ami                         = "${var.ami_id}"
```

Comment:
Maybe use https://www.terraform.io/docs/providers/aws/d/ami.html to find a default ami?

Reply:
I thought about that, but then you wouldn't be able to automate it through Concourse for example, as you wouldn't be able to pass the ami id as a variable.

Reply:
you can always override it no?

Reply:
done